ContentFiles/ElementPackages/StandardLibrary/Complex.ele

```diff
@@ -1,9 +1,53 @@
-#struct Complex(real:Num, imaginary:Num)
-#{
-#    add(a:Complex, b:Complex):Complex = Complex(a.real.add(b.real), a.imaginary.add(b.imaginary));
-#    mul(a:Complex, b:Complex):Complex = Complex(
-#            a.real.mul(b.real).sub(a.imaginary.mul(b.imaginary)),
-#            a.real.mul(b.imaginary).add(a.imaginary.mul(b.real))
-#        );
-#    norm(c:Complex):Num = c.real.mul(c.real).add(c.imaginary.mul(c.imaginary));
-#}
+struct Complex(real:Num, imaginary:Num)
+{
+    zero = Complex(0, 0)
+    unitReal = Complex(1, 0)
+    unitImaginary = Complex(0, 1)
+
+    fromPolar(r:Num, theta:Num):Complex
+    {
+        # Create a Complex number from its polar coordinates
+        # the angle theta must be in radians
+
+        x = r.mul(Num.cos(theta))
+        y = r.mul(Num.sin(theta))
+        return = Complex(x, y)
+    }
+
+    re(c:Complex):Num = c.real
+    im(c:Complex):Num = c.imaginary
```

Comment:
this and `re` aren't very useful, a user can just do Complex(1, 0).real directly, so I think it's worth removing

Reply:
The team is a little split on this, but as they are common shorthands in maths I'm okay to leave it as it is :)

I think in the future we're going to want to be more picky and maybe review this again. Adding extra shorthands is going to pollute the names in a struct for intellisense/node editors. I'm also not convinced adding shorthands because they're common in maths is a good idea.

C++ uses `real` and `imag` https://docs.julialang.org/en/v1/manual/complex-and-rational-numbers/
C# uses `real` and `imaginary` https://docs.julialang.org/en/v1/manual/complex-and-rational-numbers/
Python uses `real` and `imag` https://docs.python.org/2/library/cmath.html
Matlab uses `real` and `imag` https://docs.julialang.org/en/v1/base/numbers/#Base.Complex

Reply:
After further discussion we all agree that `real` and `imag` is the best solution :)